### PR TITLE
Report sending update

### PR DESCRIPTION
**Pull request by marcbradshaw**
Retrieve reports in batch rather than one at a time, this greatly improves performance when there are a large number of reports to be sent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,19 +147,19 @@ The remote IP that attempted sending the message. DMARC only uses this data for
 
 The domain portion of the RFC5321.RcptTo, (aka, the envelope recipient), and the bold portion in the following example:
 
-> RCPT TO:<user@**example.com**>
+> RCPT TO:&lt;user@**example.com**>
```
Comment (msimerson) [inline README.md]: why encode < here and not >?

File: README.md [hunk 2]
```diff
@@ -279,17 +279,17 @@ Qpsmtpd plugin: https://github.com/smtpd/qpsmtpd/blob/master/plugins/dmarc
...
-- Marc Bradshaw <marc@marcbradshaw.net>
-- Priyadi Iman Nurcahyo <priyadi@priyadi.net>
-- Ricardo Signes <rjbs@cpan.org>
-- Ricardo Signes <rjbs@users.noreply.github.com>
+- Benny Pedersen &lt;me@junc.eu>
+- Making GitHub Delicious. &lt;iron@waffle.io>
+- Marc Bradshaw &lt;marc@marcbradshaw.net>
+- Priyadi Iman Nurcahyo &lt;priyadi@priyadi.net>
+- Ricardo Signes &lt;rjbs@cpan.org>
+- Ricardo Signes &lt;rjbs@users.noreply.github.com>
```
Comment (msimerson) [inline README.md]: IIRC, this section is auto-generated by a dist::zilla plugin (and is thus subject to automated reversion). We might look into seeing if that plugin will URL escape for us, and if not, used sed or perl to url encode the angle brackets.

**Comment by marcbradshaw** [comment]
Yeah, looks like something has changed in dz which is doing that encoding.  It wasn't a manual change, and I usually revert README changes for that very reason.

**Comment by msimerson** [comment]
Ah well if it's DZ, then never mind my comments. :-)

**Pull request merged**